.gitattributes

```diff
@@ -0,0 +1,3 @@
+# Auto-detect text vs. binary files and ensure newlines are always LF for
```

Comment:
I will remove this file

Reply:
Same reason of .editorconfig  - standardize environmentes. After I've cloned the repo, all lines breaks were changed, beacause Git on Windows uses CRLF as default.

Reply:
This can be prevented by configuring git on windows or stopping using windows ;)